fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,56 @@
+--- Supported modes:
```

Comment:
@olgamiller right at the start, a few words on what this app is about would be very much welcome. Those not familiar with the area otherwise have no clue at all and immediately lose interest (I for example would have already left were I not a F-Droid maintainer reviewing this). I had to search some of the buzz words to make a guess: [Slow-Scan TeleVision](https://en.wikipedia.org/wiki/Slow-scan_television)?